src/interactions.rs

```diff
@@ -56,19 +57,26 @@ pub struct EditIssueBody<'a> {
 static START_BOT: &str = "<!-- TRIAGEBOT_START -->\n\n";
 static END_BOT: &str = "<!-- TRIAGEBOT_END -->";
 
+fn normalize_body(body: &str) -> String {
+    str::replace(body, "\r\n", "\n").to_string()
```

Comment:
str::replace already returns a String, so the to_string() here isn't needed.

Reply:
fixed in c500466